editor/src/main/com/mbrlabs/mundus/editor/ui/modules/dialogs/SkyboxDialog.kt

```diff
@@ -142,6 +156,22 @@ class SkyboxDialog : BaseDialog("Skybox"), ProjectChangedEvent.ProjectChangedLis
 
                 projectManager.current().currScene.setSkybox(selectBox.selected, Shaders.skyboxShader)
                 resetImages()
+                resetFields()
+            }
+        })
+
+        rotateEnabled.addListener(object : ChangeListener() {
+            override fun changed(event: ChangeEvent?, actor: Actor?) {
+                projectManager.current().currScene.skybox.isRotateEnabled = rotateEnabled.isChecked
+                addModifiedAsset()
+            }
+        })
+
+        rotateSpeed.textFieldFilter = FloatDigitsOnlyFilter(true)
+        rotateSpeed.addListener(object : ChangeListener() {
+            override fun changed(event: ChangeEvent?, actor: Actor?) {
+                projectManager.current().currScene.skybox.rotateSpeed = rotateSpeed.text.toFloat()
```

Comment:
If the rotate speed text is empty, e.g. want to change the whole value, then here is an NumberFormatException.

Reply:
Good catch, should be fixed now